test/external/external_test_datasets.py

```diff
@@ -0,0 +1,71 @@
+from extra.datasets.kits19 import iterate, preprocess
+from test.external.mlperf_unet3d.kits19 import PytTrain, PytVal
+from tinygrad.helpers import temp
+from pathlib import Path
+
+import nibabel as nib
+import numpy as np
+import os
+import random
+import tempfile
+import unittest
+
+class ExternalTestDatasets(unittest.TestCase):
+  def _set_seed(self):
+    np.random.seed(42)
+    random.seed(42)
+
+  def _create_sample(self):
+    self._set_seed()
+
+    img, lbl = np.random.rand(190, 392, 392).astype(np.float32), np.random.randint(0, high=1024, size=(190, 392, 392)).astype(np.uint8)
```

Comment:
why is it uint8 but high=1024?

Reply:
this one was accidentally left out. set it to a proper value now.